integrations/README.md

```diff
@@ -0,0 +1,3 @@
+# Choreo iPaaS examples
```

Comment:
We need to change these text too , right?

Reply:
That will be handled separately. This is just for the URL change.